docs/CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+      * `kind_detail`: `motorway`, `motorway_link`, `trunk`, `primary`, `secondary`, `tertiary`, `trunk_link`, `unclassified`, `residential`, `road`, `primary_link`, `secondary_link`, `living_street`, `service`, `pedestrian`, `track`, `cycleway`, `bridleway`, `tertiary_link`, `footway`, `steps`, `corridor
+  * **water**: Reduce the marine water labels visible at zoom 8 by limiting number of bays in the tile to the largest 10. (Issue [#1838](https://github.com/tilezen/vector-datasource/issues/1838))
+      * The kinds `bay`, `strait` and `fjord` are ranked by size and given a `kind_tile_rank` property that starts from 1 and counts up as the feature gets smaller. Note that the ranking is done on a "metatile", which means that each tile (of size 256px, 512px or other) won't necessarily contain the full range from 1 to N of `kind_tile_rank`s.
+  * **water**: Water boundary lines now indicate intermittent property (playas), matching their polygons. (Issue [#1249](https://github.com/tilezen/vector-datasource/issues/1249))
+
+  #### DOCUMENTATION CHANGES
+
+  * TODO Updated TileJSON for v1.8 schema changes.
+  * TODO Updated Layers documentation for v1.8 schema changes.
+  * Resolved some dangling TODOs from v1.7.0 documentation release.
+  * **traffic_incidents**: Add `road_closure` kind to indicate if a road has been closed, e.g. due to police presence. Thanks [@conor-ettinoffe-here](https://github.com/conor-ettinoffe-here)! (Issue [#1857](https://github.com/tilezen/vector-datasource/issues/1857))
+
+  #### INTERNAL CHANGES
+
+  * Add `all_name_variants` param to `drop_properties` function which treats `name` as if it's all `name:*` and variants. (Issue [#1847](https://github.com/tilezen/vector-datasource/issues/1847))
+  * Treat `NaN` and `Inf` as invalid to deal with bad OpenStreetMap import data. (Issue [#1903](https://github.com/tilezen/vector-datasource/issues/1903))
+  * Switch Natural Earth zooms to use actual `min_zoom` instead of `scalerank`
+  * Fix polygon merging failure related to GEOS/Shapely polygon merging routine `unary_union`. (Issue [#1848](https://github.com/tilezen/vector-datasource/issues/1848))
+  * Perform faster SQL updates by moving indexes to their own separate parallel step and add 4x sharding over `osm_id` for updates to `planet_osm_*` tables. (Issue [#1844](https://github.com/tilezen/vector-datasource/issues/1844))
+  * Move more tests to generative (from live OSM data), fix a few tests
+  * Teach the **kind** parser to handle `all` filter expressions
+  * Add `capital_alternate_viewpoint` transform
+  * Add `major_airport_detector` transform
+  * Add `unpack_viewpoint_claims` transform
+  * Add a `where` clause for filtering features based on their properties, with `base_where` and `cutting_where` parameters.
+  * Add pixel `tolerance` to polygon merging.
+  * Add `split_threshold` with default of 15,000 features to line merging.
+  * Update database indexes
```

Comment:
Do we want to add a bit more detail, i.e: which database indexes did we update, and why? (I ask mainly because I've forgotten what we did!)

Reply:
Looking back at the diff this is same as the parallelism we added that's covered in line above.

Removed in https://github.com/tilezen/vector-datasource/pull/1912/commits/ec8664f26509f052490a21e61990d5a7bf83e967.